src/main/webapp/projectContainer/Submissions.js

```diff
@@ -1,47 +1,151 @@
-import { Component } from 'react';
-import { div, hh } from 'react-hyperscript-helpers';
+import { Component, Fragment } from 'react';
+import { div, a, hh, h, button, span } from 'react-hyperscript-helpers';
 import { ProjectMigration } from '../util/ajax';
+import { Panel } from '../components/Panel';
+import { MultiTab } from "../components/MultiTab";
+import { Table } from "../components/Table";
+import { Files } from "../util/ajax";
+import _ from 'lodash';
+
+const headers =
+  [
+    { name: 'Number', value: 'number' },
+    { name: 'Description', value: 'comments' },
+    { name: 'Documents', value: 'documents' },
+    { name: 'Created', value: 'createDate' },
+  ];
+
+const styles = {
+  submissionComment: {
+    margin: '0 10px 10px 0',
+    paddingLeft: '20px',
+    width: 'calc(100% - 60px)',
+    display: 'inline-block',
+    overflow: 'visible',
+    whiteSpace: 'normal',
+    textOverflow: 'initial',
+    wordBreak: 'break-word'
+  },
+
+  addSubmission: {
+    position: 'absolute',
+    right: '30px',
+    top: '46px'
+  },
+
+  submissionCounter: {
+    marginLeft: '1em'
+  }
+};
 
 export const Submissions = hh(class Submissions extends Component {
 
   constructor(props) {
     super(props);
     this.state = {
-      content: ''
+      content: '',
+      amendments:[],
+      others: [],
+      amendmentDocuments: [],
+      otherDocuments: [],
+      submissions: {},
     };
   }
 
   componentDidMount() {
-    this.getSubmissions();
+    this.getDisplaySubmissions();
   }
 
-  getSubmissions() {
-    ProjectMigration.getSubmissions(component.projectKey).then(resp => {
+  getDocumentLink = (data) => {
+    return [component.serverURL, 'api/files-helper/get-document?id=' + data].join("/");
+  };
+
+  submissionEdit = (data) => {
+    const indexButton = a({
+      className: 'btn btn-default btn-xs pull-left link-btn',
```

Comment:
It would be nice if these were also inline styles

Reply:
I think in this case it makes no sense to put Bootstrap styles inline in the component.

Reply:
👍 since these are bootstrap styles. Relatedly, one of the goals is to move away from shared css altogether - it is very difficult to maintain and introduces a host of style bugs over time.

Reply:
Yes, I agree on that.